generator/integration_tests/generator_integration_test.cc

```diff
@@ -91,6 +91,9 @@ class GeneratorIntegrationTest
     output_path_ = google::cloud::internal::GetEnv(
                        "GOOGLE_CLOUD_CPP_GENERATOR_OUTPUT_PATH")
                        .value_or(::testing::TempDir());
+    if (output_path_.empty() || output_path_.back() != '/') {
```

Comment:
I'm not sure that an empty output path means we should write to the root directory, as opposed to the working directory.

Reply:
PTAL